src/transformers/models/gpt_bigcode/modeling_gpt_bigcode.py

```diff
@@ -342,11 +351,12 @@ class GPTBigCodeMLP(nn.Module):
     def __init__(self, intermediate_size, config):
         super().__init__()
         embed_dim = config.hidden_size
-        self.c_fc = torch.nn.Linear(embed_dim, intermediate_size)
-        self.c_proj = torch.nn.Linear(intermediate_size, embed_dim)
+        self.c_fc = Linear(embed_dim, intermediate_size)
+        self.c_proj = Linear(intermediate_size, embed_dim)
         self.act = ACT2FN[config.activation_function]
-        self.dropout = nn.Dropout(config.resid_pdrop)
+        self.dropout = Dropout(config.resid_pdrop)
 
+    # Copied from transformers.models.gpt2.modeling_gpt2.GPT2MLP.forward
```

Comment:
@younesbelkada Is this safe, given that we modify the layers?

Reply:
Yes! Otherwise some checks would fail, here they are passing as we didn't touched anything on the forward function

Reply:
I mean safe with respect to potential future changes in GPT2...

Reply:
I don't expect breakign changes to happen in GPT2, and in the worst case we could always remove that statement in the future! :D